src/command_modules/azure-cli-iotcentral/azure/cli/command_modules/iotcentral/custom.py

```diff
@@ -66,6 +67,16 @@ def _check_name_availability(client, app_name):
         raise CLIError(name_availability.message)
 
 
+def _check_subdomain_availability(client, app_subdomain):
+    """Search the current subscription for an app with the given subdomain.
+    :param object client: IoTCentralClient
+    :param str app_subdomain: App subdomain to search for
+    """
+    subdomain_availability = client.apps.check_subdomain_availability(app_subdomain)
+    if subdomain_availability is not None and not subdomain_availability.name_available:
```

Comment:
When would `subdomain_availability` be None?

Reply:
I was following the same pattern as the existing method for `check_name_availability`. I guess this shouldn't ever be `None` I can remove this if you'd like.

Reply:
applied 😃 .